lib/rules/id-ad-disabled.ts

```diff
@@ -16,7 +16,7 @@ export class IdAdDisabled extends BaseRule {
           let {id = []} = attrs;
           id = (id[0] || {});
 
-          if (/(^|[-_])(ad?|sponsors?)([-_]|$)/i.test((id.content || ''))) {
+          if (/(^|[-_])(ad|sponsor(s)?)([-_]|$)/i.test((id.content || ''))) {
```

Comment:
Changed this in my branches as well...this needs to be changed in class-ad-disabled also.

```suggestion
          if (/(^|[-_])(ads?|sponsors?)([-_]|$)/i.test((id.content || ''))) {
```